sdk/maps/maps-route-rest/README.md

```diff
@@ -0,0 +1,190 @@
+  queryParameters: {
+    query: toColonDelimitedLatLonString([
+      [51.368752, -0.118332],
+      [41.385426, -0.128929],
+    ]),
+    vehicleWidthInMeters: 2,
+    vehicleHeightInMeters: 2,
+    vehicleLoadType: "USHazmatClass1",
+    travelMode: "truck",
+    isCommercialVehicle: true,
+  },
+});
+```
+
+### Calculate and optimize a multi-stop route
+
+Azure Maps currently provides two forms of route optimizations:
+
+- Optimizations based on the requested route type, without changing the order of waypoints. You can find the [supported route types here](https://docs.microsoft.com/rest/api/maps/route/post-route-directions?tabs=HTTP#routetype).
+- Traveling salesman optimization, which changes the order of the waypoints to obtain the best order to visit each stop
+
+For multi-stop routing, up to 150 waypoints may be specified in a single route request. The starting and ending coordinate locations can be the same, as would be the case with a round trip. But you need to provide at least one additional waypoint to make the route calculation. Waypoints can be added to the query in-between the origin and destination coordinates.
+
+If you want to optimize the best order to visit the given waypoints, then you need to specify `computeBestWaypointOrder=true`. This scenario is also known as the traveling salesman optimization problem.
+
+```javascript
+const routeDirectionsResult = await client.path("/route/directions/{format}", "json").get({
+  queryParameters: {
+    query: toColonDelimitedLatLonString([
+      [51.368752, -0.118332],
+      [41.385426, -0.128929],
+    ]),
+    computeBestWaypointOrder: true,
+  },
+});
+```
+
+## Troubleshooting
+
+### Logging
+
+Enabling logging may help uncover useful information about failures. In order to see a log of HTTP requests and responses, set the `AZURE_LOG_LEVEL` environment variable to `info`. Alternatively, logging can be enabled at runtime by calling `setLogLevel` in the `@azure/logger`:
+
+```javascript
+import { setLogLevel } from "@azure/logger";
+
+setLogLevel("info");
+```
+
+For more detailed instructions on how to enable logs, you can look at the [@azure/logger package docs](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/core/logger).
+
+## Next steps
+
+Please take a look at the [samples](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/maps/maps-route/samples) directory for detailed examples on how to use this library.
+
+## Contributing
+
+If you'd like to contribute to this library, please read the [contributing guide](https://github.com/Azure/azure-sdk-for-js/blob/master/CONTRIBUTING.md) to learn more about how to build and test the code.
+
+## Related projects
+
+- [Microsoft Azure SDK for JavaScript](https://github.com/Azure/azure-sdk-for-js)
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-js%2Fsdk%2Fmaps%2Fmaps-route%2FREADME.png)
```

Comment:
`maps-route` -> `maps-route-rest`

Reply:
Fixed in 96781dc3df277391148097eb60c21688978091e5 Thanks!